README.md

```diff
@@ -28,7 +28,26 @@ The call would return `['foo', 'boo', 'bar']`.
 Feel free to use other data structures, but you'll have to change the test code
 accordingly.
 
-## Runtime Analysis
+# Runtime Analysis
 
 What is the worst-case big $\Theta$ complexity of your implementation? Add your
 answer, including your reasoning, to this markdown file.
+
+## My Runtime Analysis
+
+When my augmentingPath() algorithm runs, it first checks if the start node is already the target node. If so, it will run in constant time. 
+
+Otherwise, the recursive findPath() method is called. The same check occurs at the beginning of this method, if the current node matches the end node, this is the base case and will return the target node when found. If the target node was not yet found, the current node is added to the visited list, and reaches a for loop which iterates over the edges of the current node, which occurs $|E|$ times for the current node. In the for loop, if there is a child node that has not been visited yet, recursively call findPath() setting the child node to the current node.
+
+The worst case scenerio for this algorithm is when a valid augmented path is the last path that is check within the graph, meaning it will iterate through all nodes within the graph. This will execute $|V|$ times. 
+
+Since every node is checked, and all the edges are iterated through as well (can be thought as a nested operation), the final time complexity would be $\Theta(|V| * |E|)$.
```

Comment:
Are you really iterating over *all* edges for *all* nodes?

Reply:
I am at least checking until I find a valid next node to traverse to. And since this is the worst case scenario, it would seem that you would need to go through every node (unless the node were already visited), checking every edge to ensure you have exhausted all possible paths.

Reply:
Sure, but you're multiplying instead of adding those in the complexity expression. That would suggest that you're exploring all edges for each node.

Reply:
I believe I understand now. I will reread my analysis to ensure it follows this, but essentially I just need to change that multiplication into addition (and maybe some of the wording) since I am not truly traversing every edge?

Reply:
That sounds like it's going in the right direction.